avaframe/out3SimpPlot/plotSettings.py

```diff
@@ -0,0 +1,50 @@
+"""
+    Plot settings for output figures
+
+    This file is part of Avaframe.
+"""
+
+import seaborn as sns
+from matplotlib import cm
+import copy
+import matplotlib
+
+# define figure dimentions
+figW = 12
+figH = 8
+# define figure resolution
+figReso = 150
+# define lines and marker properties
+lw = 2
+ms = 10
+markers = ['+', 'o', 'x', '*', 's', 'd', '^', 'v', '>', '<', 'p', 'h', '.',
+           '^', 'v', '>', '<', 'p', 'h', '.']
+# font size
+fs = 10
+
+# define seaborn style and color maps
+sns.set(font_scale = 1)
+sns.set_style("dark", {'axes.edgecolor': '.8', 'xtick.bottom': True, 'ytick.left': True})
+# sns.set_style("dark", {'xtick.bottom': True,
+#  'ytick.left': True})
+
+# hell white/green to dark blue
+cmapGB = sns.cubehelix_palette(8, start=.5, rot=-.75, as_cmap=True)
+cmapGB.set_bad(color='k')
+# hell pink to dark purple
+cmapPP = sns.cubehelix_palette(8, as_cmap=True)
+cmapPP.set_bad(color='k')
+
+cmapjet = copy.copy(matplotlib.cm.jet)
```

Comment:
why do we need Jet?

Reply:
it is just defined in case we want to use it, bun not used